w3-blob.md

```diff
@@ -162,7 +166,13 @@ Shows an example receipt for the above `/space/content/add/blob` capability invo
         "cmd": "/http/put",
         "sub": "did:key:zMh...der", // <-- Ed299.. derived key from content multihash
         "args": {
-          "content": { "/": { "bytes": "mEi...sfKg" } },
+          "body": {
+            // multihash of the blob as byte array
+            "content": { "/": { "bytes": "mEi...sfKg" } },
+            // size of the blob in bytes
+            "size": 2_097_152,
+          }
+          // TODO: this should now be updated?
           "address": { "await/ok": { "/": "bafy...alloc" } },
```

Comment:
@Gozala this ended up changing below to use `url` and `headers` https://github.com/web3-storage/specs/blob/main/w3-blob.md#put-blob-capability-schema

I honestly liked more the previous alignment here with address, but happy to change this if you have strong preference to use `url` and `header` top level

Reply:
I assume if so, we would like to have something along the lines of 

`'ucan/await': ['.out.ok.claim', acceptfx.link()],`

Reply:
I end up updating it here https://github.com/web3-storage/specs/pull/118/files last week but forgot to submit a PR for the review. I do not feel very strongly about it, however I find `{ url, headers, body }` to be more obvious choice for `http/put` than `{ address, body }`.